app/Controllers/userController.php

```diff
@@ -211,9 +211,20 @@ public function manageAction() {
 		}
 	}
 
-	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfig = array(), $insertDefaultFeeds = true) {
-		if (!is_array($userConfig)) {
-			$userConfig = array();
+	public static function createUser($new_user_name, $email, $passwordPlain, $apiPasswordPlain, $userConfigOverride = array(), $insertDefaultFeeds = true) {
+		$userConfig = array();
+
+		$customUserConfigPath = join_path(DATA_PATH, 'config-user.custom.php');
+		if (file_exists($customUserConfigPath)) {
+			$customUserConfig = include($customUserConfigPath);
+		}
+
+		if (is_array($customUserConfig)) {
```

Comment:
Ups, undefined variable here @marienfressinaud  . Fixing as part of https://github.com/FreshRSS/FreshRSS/pull/2522

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2522/commits/dda6097b6c4c32426e18fdf495a715fdda881234